tests/python/unittest/test_codegen_x86.py

```diff
@@ -0,0 +1,59 @@
+import tvm
+import re
+import os
+import ctypes
+
+def test_fp16_to_fp32_with_f16c():
+    target = 'llvm -mcpu=core-avx2 -mattr=+f16c'
```

Comment:
So the `mattr` flag needs to be set by the users?

Reply:
I've fixed this to query the TargetMachine directly, so no changes are needed (assuming users are using the existing -mcpu=core-avx2, -mcpu=skylake-avx512, etc)